README.md

```diff
@@ -100,7 +100,7 @@ When a command is actually run, it can access these arguments in the context pro
 ## Parsing user input
 So, we've registered some commands and now we're ready to take in user input. If you're in a rush, you can just call `dispatcher.execute("foo 123", source)` and call it a day.
 
-The result of `execute` is an integer returned by the command it evaluated. Its meaning varies depending on command, and typically will not be useful to programmers.
+The result of `execute` is an integer returned by the command it evaluated. Its meaning varies depending on the command, and typically will not be useful to programmers.
```

Comment:
should be `a commands it has evaluated`

Reply:
It should be the not a

Reply:
Nope. Here is an example:
```
I ate an apple. The apple was sweet.
```
Similarly, these sentences below are also valid:
```
The method have evaluated a command. The command defines the meaning of the integer.
```
@boomboompower Thus, I disagree with your proposal.

Reply:
"The method have evaluated a command." does not sound valid to me.

Reply:
Oh, it was just a minor issue I've accidentally introduced.

Reply:
Fixed. @Pokechu22

Reply:
I think I agree with @boomboompower, it doesn't sound right to me without the "the". However, lets forget about that since I think this will work.

> The result of `execute` is an integer that was returned from an evaluated command. The meaning of this integer depends on the command, and will typically not be useful to programmers.